gnmi_server/server_test.go

```diff
@@ -107,7 +107,7 @@ func createServer(t *testing.T, port int64) *Server {
 	}
 
 	opts := []grpc.ServerOption{grpc.Creds(credentials.NewTLS(tlsCfg))}
-	cfg := &Config{Port: port, EnableTranslibWrite: true, EnableNativeWrite: true}
+	cfg := &Config{Port: port, EnableTranslibWrite: true, EnableNativeWrite: true, Threshold: 100}
```

Comment:
>100 [](http://example.com/codeflow?start=91&length=3)

If there is no ConfigDB entry, you will use default value (100). Maybe there is use case that user want to have static default value changed to other values. Is it better to move this default value to a config file inside telemetry container?